### DISCUSSION TITLE
Share your plans and ideas with the Azure SaaS Development Kit

### DISCUSSION THREAD

**Discussion by 1iveowl** [Azure/azure-saas#219] [Show and tell]
What are your plans with the Azure SaaS Development Kit? Please share your thoughts, ideas and aspirations.

**Comment by yogjai** [comment]
Hi, We are developing a Multitenant B2B SaaS Solution and we need some support to customize the identity framework according to our application requirement. We basically want to authenticate every user from every identity provider. Can you please help?
